src/common/src/test/scala/common/ProcessOSMTest.scala

```diff
@@ -0,0 +1,38 @@
+package osmesa.common
+
+import geotrellis.vector._
+import geotrellis.geotools._
+// import geotrellis.geomesa.geotools. _
+import org.locationtech.geomesa.spark.jts._
+import org.scalatest._
+import org.geotools.feature.simple.SimpleFeatureImpl
+import org.apache.spark.sql._
+import org.apache.spark.sql.types._
+import org.apache.spark.sql.functions._
```

Comment:
Can you clean up the imports in this? Many of these are unused.

Reply:
:+1: